### ISSUE TITLE
Sway consistently crashes with neovim

### ISSUE THREAD

**Issue by tasn** [swaywm/sway#4563]
* Sway Version: sway version 1.2 (Aug 29 2019, Arch Linux)
* Debug Log: https://pastebin.com/qzRuPxSX
* Configuration File: I'm using my own configuration. Been spending over an hour recovering from this and trying to gdb attach and etc.

Reproduce:
1. Run neovim like this: `neovim ~/`
2. Move the cursor a few lines down.
3. Crash.

Note: this doesn't seem related to #4517 because copy/paste works just fine.

* Stack Trace, if sway crashes:
```
                
                Stack trace of thread 4697:
                #0  0x00007f1e386d5b58 wlr_seat_keyboard_enter (libwlroots.so.3)
                #1  0x000055e824343368 n/a (sway)
                #2  0x000055e82434326b n/a (sway)
                #3  0x000055e824344ffd n/a (sway)
                #4  0x000055e8243450db n/a (sway)
                #5  0x00007f1e386fccfe n/a (libwlroots.so.3)
                #6  0x00007f1e386fccfe n/a (libwlroots.so.3)
                #7  0x00007f1e386c633a n/a (libwlroots.so.3)
                #8  0x00007f1e386c510c n/a (libwlroots.so.3)
                #9  0x00007f1e387417f2 wl_event_loop_dispatch (libwayland-server.so.0)
                #10 0x00007f1e3874039c wl_display_run (libwayland-server.so.0)
                #11 0x000055e824318592 n/a (sway)
                #12 0x00007f1e38d42ee3 __libc_start_main (libc.so.6)
                #13 0x000055e8243188be n/a (sway)
                
                Stack trace of thread 4722:
                #0  0x00007f1e38605415 pthread_cond_wait@@GLIBC_2.3.2 (libpthread.so.0)
                #1  0x00007f1e36690f5c n/a (i965_dri.so)
                #2  0x00007f1e36690b58 n/a (i965_dri.so)
                #3  0x00007f1e385ff57f start_thread (libpthread.so.0)
                #4  0x00007f1e38e180e3 __clone (libc.so.6)
```

**Comment by tasn** [comment]
@emersion, let me first find a time to test the fix for #4517 and then I can try getting sway with debug symbols.

**Comment by tasn** [comment]
Indeed fixed with latest git. Thanks and sorry for the noise.
